src/components/List/List.stories.tsx

```diff
@@ -55,6 +67,31 @@ list.parameters = {
   },
 };
 
+const DefinitionListTemplate = ({ items, ...args }) => (
+  <List type={ListTypes.Definition} {...args}>
+    {items.map((item, i) => (
+      <React.Fragment key={i}>
```

Comment:
This renders something like 

```
<React.Fragment>
  <dt></dt><dd><dd>
</React.Fragment>
<React.Fragment>
  <dt></dt><dd><dd>
</React.Fragment>
...
```

You can get away from the fragment by returning an array. So you can return `[<dt>...</dt>, <dd>...</dd>]` and it'll render:

```
  <dt></dt><dd><dd>
  <dt></dt><dd><dd>
  ...
```

Reply:
Nice!!!  That _did_ work!  Very cool.